web/src/app/not-found.tsx

```diff
@@ -1,10 +1,12 @@
 import Link from "next/link";
+import "./layout.css";
+import "./not-found.css";
```

Comment:
Ideally, you wouldn't have to import `layout.css` in any file other than `layout.tsx` because every other page inherits the top level layout.

Reply:
Indeed, but it didn't seem like the not-found was picking it up, not sure why it didn't seem like it was inheriting. Will test again locally and see if I'm still getting the issue

Reply:
yep seems to be fine now